rules/robot_players.tex

```diff
@@ -49,24 +44,24 @@ \subsection{Team Markers}
 Teams may design and manufacture their own jerseys in any color (multi and many color jerseys are acceptable), but must follow these guidelines:
 \begin{itemize}
 \item Jerseys should be the \textbf{tank top} style used at RoboCup 2013/2014 and should cover approximately the same areas of the robot as shown in \cref{fig:nao_markers}. The torso LED must be clearly visible. Jerseys may include the sonar panel used in the 2013/2014 jerseys, although this is not required. Jerseys may not cover the shoulders of the robots.
-\item Jerseys must have a primary color that comprises at least \qty{70}{\percent} of the jersey.
+\item Jerseys must have a primary color that comprises more than half of the jersey.  The primary color must be recognizable from the front and back.
 \item Jerseys should not contain distractors, such as large pictures of SPL balls or white stripes on green jerseys.
-\item All players on a team must wear identical jerseys, including the goalkeeper.
 \item A team must wear the jerseys that it starts a game in for the entire game.
-\item Jersey material must be non-reflecting, non-shiny, and non-textured.  Material that is glittery is also not appropriate.
-\item Jerseys should be numbered 1-6 on both sides.  The numbers must be large and {\bf easily} recognized by humans.
-\item Teams must have two sets of jerseys that are significantly different in terms of their primary color.
+\item Jersey material must be non-reflecting and non-shiny.  Material that is glittery is also not appropriate.  Jerseys may be manufactured from fabric and fine mesh.
+\item Jerseys should be numbered 1-{\MaxJerseyNumber} on both sides.  The numbers must be large and {\bf easily} recognized by humans.
+\item Teams must have two sets of jerseys for \emph{field players} that are significantly different in terms of their primary color.
+\item Teams must have two jerseys for their \emph{goalkeeper} that are significantly different in terms of their primary color.  One of the primary colors must be different from both primary colors used for \emph{field players}.
```

Comment:
My understanding is:

- Two teams must have field player colours which are sufficiently different from each other during a game
- Two teams may have goalies who play with the same or very similar colours to each other, but the colour must be different from their respective field players. (At least this point was raised in the original issue text and I would agree with it since it matches the FIFA rules - Law 4-3.)
- If you accept the two points above, then I don't believe it is strictly necessary to have any additional jerseys since the goalie could wear the away colour when the team wears home, and could wear home when the team wears away.

Reply:
But if two teams have the same set of home and away colors, then you would have both goalkeepers in the opponent's color. That is not nice. Therefore, we said it should be a third color. But you are right the goalkeepers could wear the same color